src/winspy.vcxproj

```diff
@@ -75,7 +94,7 @@
     <Link>
       <SubSystem>Windows</SubSystem>
       <DelayLoadDLLs>uxtheme.dll;windowscodecs.dll</DelayLoadDLLs>
-      <AdditionalDependencies>psapi.lib;version.lib;uxtheme.lib;windowscodecs.lib;comctl32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>psapi.lib;version.lib;uxtheme.lib;windowscodecs.lib;comctl32.lib;gdi32.lib;Advapi32.lib;Shell32.lib;Ole32.lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
Is this change necessary?

Reply:
I don't know. I compiled with VS 2017 and it complained me for some unresolved functions.